samples/Options.MultiTenant/TenantResolver.cs

```diff
@@ -0,0 +1,14 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace AuthSamples.Options.MultiTenant
+{
+    public class TenantResolver
```

Comment:
Interface rather than class? That way we could push a database into it, so people can map from config in a database

Reply:
So this isn't something we are shipping, this is just a sample, so there's not really much upside for an interface in the sample when they would just modify this to take any dependencies needed, if they wanted it to use a DbContext, they could just make it Scoped, although that would affect the lifetimes of the other things too